contactSalon.css

```diff
@@ -0,0 +1,85 @@
+body {
+  font-family: 'Roboto', sans-serif;
+}
+
+header{
+  display: grid;
+  grid-template-columns: 1fr 1fr;
```

Comment:
compared to the template, the logo isn't in the right place. i wonder if you could have added another column.
`grid-template-columns: 100px 1fr 1fr;`
then move the `logo { grid-column: 2/3;}`
and `nav {grid-column: 3/-1;}`